index.bs

```diff
@@ -184,12 +179,15 @@ Integration with HTML {#html}
 
 This specification integrates with the [[!HTML]] specification by patching the algorithms below:
 
-Document object initialization {#document-init}
+Navigation fetch {#navigation-fetch}
+----------
+
+At [=process a navigate fetch=], after step 13.4 call [$retrieve the Client Hints set$] with <var ignore>currentURL's origin</var> as input.
+
+Navigation response {#navigation-response}
 ----------
 
-At <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#initialise-the-document-object">Create and initialize a Document object</a>,
-after step 11, starting with "Initialize a Document's CSP list",
-call [$initialize the Client Hints set$] with <var ignore>document</var>'s [=relevant settings object=] and |response| as inputs.
+At [=process a navigate response=], after step 7 call [$update the Client Hints set$] with the [=relevant settings object=], <var ignore>currentURL's origin</var>, and |response| as inputs.
```

Comment:
"process  navigate response" doesn't seem to have a `currentURL` variable

Reply:
Can you expand on the origin discrepency that this is fixing? (I'm assuming that's the reason to pass the origin separately)

Reply:
Reverted this bit to the way it was before, this fix wasn't intended to resolve an issue with getting the origin from the settings object, just an issue with starting the process at `Document object initialization` instead of setting initial hints at `process navigate fetch` and updating the list at `process navigate response`.